### PR TITLE
feat(task): additional prepend methods

### PR DESCRIPTION
**Pull request by hoegertn**
Allow to prepend not only exec tasks but also spawn and say tasks.

### CODE REVIEW

**Comment by hoegertn** [comment]
@projen rebuild

**Review by Chriscbr — APPROVED** [review]
LGTM!

**Comment by Chriscbr** [comment]
@Mergifyio refresh

File: src/tasks/task.ts [hunk 1]
```diff
@@ -92,20 +92,52 @@ export class Task {
   /**
    * Adds a command at the beginning of the task.
    * @param shell The command to add.
+   *
+   * @deprecated use prependExec()
```
Comment (eladb) [inline src/tasks/task.ts:96]:
```suggestion
   * @deprecated use `prependExec()`
```

**Pull request merged**